cp3pt0-deployment/setup_singleton.sh

```diff
@@ -194,6 +200,7 @@ EOF
     create_operator_group "ibm-licensing-operator-app" "${LICENSING_NAMESPACE}" "$target"
     create_subscription "ibm-licensing-operator-app" "${LICENSING_NAMESPACE}" "$CHANNEL" "ibm-licensing-operator-app" "${LICENSING_SOURCE}" "${SOURCE_NS}" "${INSTALL_MODE}"
     wait_for_operator "${LICENSING_NAMESPACE}" "ibm-licensing-operator"
+    # accept_license "ibmlicensing" ""
```

Comment:
should be able to uncomment this, doesn't matter if licensing hasn't implemented it yet, we can still patch the CR with it.